crates/server/README.md

```diff
@@ -0,0 +1,257 @@
+# Calimero Server
+
+- [Introduction](#introduction)
+    - [Admin API](#1admin-api)
+    - [JSON rpc](#2json-rpc)
+    - [Websocket](#3websocket)
+- [Calimero Server Workflows](#calimero-server-workflows)
+    - [Admin API Workflow](#workflow-for-admin-dashboard)
+    - [JSON rpc Workflow](#workflow-for-p2p-web-applications-and-json-rpc)
+    - [Websocket Workflow](#workflow-for-p2p-web-application-and-websocket)
+- [Admin API endpoints](#admin-api-endpoints)
+    - [Protected Routes](#protected-routes)
+    - [Unprotected Routes](#unprotected-routes)
+- [JSON rpc endpoints](#json-rpc-endpoints)
+- [Websocket endpoints](#websocket-endpoints)
+- [Examples](#examples)
+
+## Introduction
+
+The Calimero Server is a server component of the Calimero Node that has crucial part in
```

Comment:
Node server is a component in Node that facilitates node administration and enables communication with the logic of application (loaded wasm) in participating contexts.